classes/Provider/TwigServiceProvider.php

```diff
@@ -38,7 +38,7 @@ public function register(Application $app)
         if ($enddate->format('H:i:s') == '00:00:00') {
```

Comment:
I'm going to merge this into the Silex 2.0 branch and clean up any of the CS errors